src/components/Search.js

```diff
@@ -12,11 +12,13 @@ export default class Search extends React.Component {
   }
 
   onChange(e) {
-    var value = e.target.value;
+    const value = e.target.value,
+          keywords = e.target.value.toLowerCase().split(' ');
+
     this.props.onFilteredData(_.filter(this.props.data, function (row) {
-      return _.find(_.values(row), function (val) {
-        return (val.toString().toLowerCase()).indexOf(value.toLowerCase()) !== -1;
-      });
+      const values = _.values(row);
+      const words = _.join(values, ' ').toLowerCase();
```

Comment:
Yep. that should all be sorted. Both the words and the keywords are lowercase.

I think it should work fine because this isn't doing a comparison of two arrays.

It's comparing a long string of all the words in the row with an array of keywords - everything lowercase.

fyi - this PR has been merged...